recipes/visualpic/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set version = "0.5.1" %}
+
+package:
+  name: visualpic
+  version: {{ version }}
+
+source:
+  url: https://github.com/AngelFP/VisualPIC/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 5ecbe92fac5ea5148d901438ab5d734ce467653becbf0cf9f64baa8a86577f14
+
+build:
+  number: 0
+  noarch: python
```

This should fix the Windows failure:

```suggestion
  noarch: python
  entry_points:
      - vpic = visualpic.cli.vpic:vpic
      - vpic3d = visualpic.cli.vpic3d:vpic3d
```